build/publish.proj

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+
+  <!--
+    ============================================================
+    This is for orchestrated build scenarios. The NuGet.Client-Orchestrated build definition builds this file directly.
+    ============================================================
+  -->
+
+  <Import Project="$(BlobPublishTargetsPath)" />
+  <Import Project="$(BuildCommonDirectory)common.targets" />
+
+  <ItemGroup>
+    <ItemsToPush Include="$(NupkgOutputDirectory)*.nupkg" />
+  </ItemGroup>
+
+  <Target Name="Build" Condition=" '$(BuildRTM)' == 'false' AND Exists($(NupkgOutputDirectory))">
```

Comment:
I'd use '$(BuildRTM)' != 'true' as it's safer.

Reply:
I assume the target name is mandated by OrchBuild reqs?

Should the target actually fail when the nupkgs directory is not there, rather than not execute at all?

Reply:
target name is build because the project file has is intended to be used for one purpose.

If this target cannot be completed then the build should fail as there will be no artifacts downstream.